### PR TITLE
Modernise package configuration

### PR DESCRIPTION
**Pull request by mpougin**
I tried to migrate the setup files as done by @chrisjsewell in https://github.com/lsmo-epfl/aiida-lsmo/pull/101

### CODE REVIEW

**Comment by mpougin** [comment]
oh, it actually works fine, had some installation issues locally. Is it okay to merge?

**Comment by mpougin** [comment]
what is this `continuous-integration/travis-ci`? I think it can be removed?

**Comment by chrisjsewell** [comment]
> what is this `continuous-integration/travis-ci`? I think it can be removed?

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+    "aiida-testing-dev==0.1.0.dev1",
+    "pytest-cov~=2.7",
+    "pytest~=6.0",
+]
+
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.packages.find]
+namespaces = false
```
Comment (chrisjsewell) [inline pyproject.toml:67]:
```suggestion
```

you don't need these as you're using flit instead of setuptools

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+    "Development Status :: 5 - Production/Stable",
+]
+dependencies = [
+    "aiida-core>=1.1.0,<3",
+    "six",
+    "voluptuous",
+]
+
+[project.license]
+text = "MIT License"
```
Comment (chrisjsewell) [inline pyproject.toml:30]:
```suggestion
file = "LICENSE"
```

**Comment by mpougin** [comment]
> > what is this `continuous-integration/travis-ci`? I think it can be removed?
> 
> https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

So I can remove the bottom one?
Screenshot from 2022-12-06 14-22-42

**Comment by mpougin** [comment]
and thanks a lot for your feedback @chrisjsewell

**Comment by chrisjsewell** [comment]
Yeh, then search for the new (gh actions) ones and add them instead

**Pull request merged**